exporters/elasticsearch-exporter/src/test/java/io/camunda/zeebe/exporter/TestSupport.java

```diff
@@ -32,11 +34,20 @@ private TestSupport() {}
    *
    * <p>Additionally, security is explicitly disabled to avoid having tons of warning printed out.
    */
+  @SuppressWarnings("resource")
   static ElasticsearchContainer createDefaultContainer() {
     return new ElasticsearchContainer(ELASTIC_IMAGE)
-        .withEnv("ES_JAVA_OPTS", "-Xms256m -Xmx512m -XX:MaxDirectMemorySize=536870912")
+        // use JVM option files to avoid overwriting default options set by the ES container class
+        .withClasspathResourceMapping(
+            "elasticsearch-fast-startup.options",
```

Comment:
:thought_balloon: ES has this nice way to allow users to customize JVM options without overwriting each other by using a folder with JVM option files.